stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/application/App.kt

```diff
@@ -21,7 +21,7 @@ class App : Application() {
         Coil.setImageLoader(
             ImageLoader.Builder(this).componentRegistry {
                 if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
-                    add(ImageDecoderDecoder())
+                    add(ImageDecoderDecoder(this@App))
```

Comment:
The same should be done in `StreamImageLoaderFactory` and `ImageLoaderFactoryInitializer`